### ISSUE TITLE
A11y: Convey how the promotion information relates to entering a discount code via Assistive Technology

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5210]
Information a sighted person would mentally group needs to be programmatically reinforced for Assistive Technology.
[https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html|https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html|smart-link] 

If you are using VoiceOver, and you enter a valid discount code with a screen reader, you get a message when the code was added successfully (great!). The problem is that when you submit, suddenly, the system will make changes which are not announced or otherwise conveyed. Specifically, it will add something (the promotion) to your cart that: 

# Wasn’t there before
# Increases the item count number
# The close button identifies as a product (with VoiceOver)
# Can be called anything (because promotions can be named anything)

Visually, you can make those connections because our brains will mentally group them. Audibly, this does not work. There’s nothing that announces that this new thing (is it an item? is it a product?) in your cart is a result of your discount code because there’s nothing to audibly identify the new ‘item’ as the promotion, and the only context that currently does exists suggests it’s a product (I didn’t add another product!).

We need to figure out a way to programmatically, audibly reinforce that the promotion:

# is related to that discount code(Promo Code ‘AB1HWTY’ was successfully added. Discount ‘IHEARTPUPPIES’ has been added to your cart')
# is a discount/promotion/the thing with the negative amount

This also needs to be audibly reinforced when the discount is removed.

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-38716**



[created from NEXT-38716, comment 505554]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.9.0**

 This issue will be / was solved in Shopware version ***6.6.9.0***.



[created from NEXT-38716, comment 517956]
